examples/chem-sync-local-flask/local_app/benchling_app/handler.py

```diff
@@ -31,6 +35,9 @@ def handle_webhook(webhook_dict: dict[str, Any]) -> None:
             render_search_canvas(app, webhook.message)
         elif isinstance(webhook.message, CanvasInteractionWebhookV2):
             route_interaction_webhook(app, webhook.message)
+        elif isinstance(webhook.message, CanvasCreatedWebhookV2Beta):
+            canvas_id = webhook_dict["message"]["canvasId"]
+            render_search_canvas_for_created_canvas(app, canvas_id)
```

Comment:
niceee... with`CanvasCreatedWebhookV2Beta` -- we can actually do something a little more ergonomic here
```suggestion
        elif isinstance(webhook.message, CanvasCreatedWebhookV2Beta):
            render_search_canvas_for_created_canvas(app, webhook.message.canvas_id)
```